conf/solr/9.3.0/schema.xml

```diff
@@ -229,6 +229,9 @@
 
     <!-- incomplete datasets issue 8822 -->
     <field name="datasetValid" type="boolean" stored="true" indexed="true" multiValued="false"/>
+
+    <!-- license searchable and facetable issues 9060 and 7482 -->
+    <field name="license" type="string" stored="true" indexed="true" multiValued="false"/>
```

Comment:
I struggle with where or even if we should document a new search field like this. I guess `license` already mentioned in the release note snippet. That's probably good enough and in line with what we've always done. Some day it might be nice to ask the Dataverse API for a list of fields you can search on, including these that aren't coming from metadata blocks (we have a API to list these).

Reply:
I also mentionned the PR in this issue for when we get to add the facets in the guide not to forget it on that side (even if reruning the command should make license appear) :
- #9515